scripts/HPC_chatbot_preprocessor/parsed_mds/generic/best_practices/best_practices_paragraph_1.txt

```diff
@@ -0,0 +1,39 @@
+Best Practices { #ch:best-practices}
```

Comment:
cleanup these references from the txt, and make them work? (these look like old latex refs, not really markdown stuff)